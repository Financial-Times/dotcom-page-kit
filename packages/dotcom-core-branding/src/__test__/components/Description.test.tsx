import React from 'react'
import renderer from 'react-test-renderer'
import Subject from '../../components/Description'

describe('dotcom-ui-shell/src/components/Title', () => {
  it('renders the document description when description prop is passed', () => {
    const tree = renderer.create(<Subject description="test description" />).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('renders an empty description when description prop is not passed', () => {
    const tree = renderer.create(<Subject />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
