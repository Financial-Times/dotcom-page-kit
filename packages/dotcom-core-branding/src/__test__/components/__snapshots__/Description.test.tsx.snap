// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dotcom-ui-shell/src/components/Title renders an empty description when description prop is not passed 1`] = `
<meta
  content="News, analysis and comment from the Financial Times, the worldÊ¼s leading global business publication"
  name="description"
/>
`;

exports[`dotcom-ui-shell/src/components/Title renders the document description when description prop is passed 1`] = `
<meta
  content="test description"
  name="description"
/>
`;
